src/Chr.Avro/Abstract/EnumSchemaBuilderCase.cs

```diff
@@ -96,6 +101,16 @@ public virtual SchemaBuilderCaseResult BuildSchema(Type type, SchemaBuilderConte
             }
         }
 
+        private string? GetDefaultValue(Type type, IEnumerable<MemberInfo> enumMembers)
+        {
+            var enumDefaultValue = type.GetAttribute<DefaultValueAttribute>()?.Value;
+            if (enumDefaultValue is null)
+                return null;
+            var matchedMember = enumMembers
+                .Single(member => member.Name == enumDefaultValue.ToString());
```

Comment:
What do you think of [`Enum.GetName`](https://learn.microsoft.com/en-us/dotnet/api/system.enum.getname) here instead? That would force the default value to be a member of the enum, which I think is what we'd want for now:

```suggestion
            var matchedMember = enumMembers
                .Single(member => member.Name == Enum.GetName(type, enumDefaultValue));
```

Reply:
Didn't think of that, nice one